src/main/java/beans/menus/EntityListResponse.java

```diff
@@ -214,16 +216,16 @@ private Entity processEntity(TreeReference entity, EntityScreen screen, Evaluati
         return ret;
     }
 
-    private void processStyles(Detail detail) {
-        DetailField[] fields = detail.getFields();
-        styles = new Style[fields.length];
-        int i = 0;
-        for (DetailField field : fields) {
-            Style style = new Style(field);
-            styles[i] = style;
-            i++;
+        private void processStyles(Detail detail) {
+            DetailField[] fields = detail.getFields();
+            styles = new Style[fields.length];
+            int i = 0;
+            for (DetailField field : fields) {
+                Style style = new Style(field);
+                styles[i] = style;
+                i++;
+            }
```

Comment:
Definitely, I am actually planning a bigger refactor to come. since the notion of displaying a case in a certain way can be contained in multiple screens, should break it out into a separate class